modules/ingest-service-impl/src/main/java/org/opencastproject/ingest/endpoint/IngestRestService.java

```diff
@@ -545,7 +545,7 @@ protected Response addMediaPackageElement(HttpServletRequest request, MediaPacka
       }
       return Response.ok(MediaPackageParser.getAsXml(mp)).build();
     } catch (Exception e) {
-      logger.warn(e.getMessage(), e);
+      logger.warn("Unable to add mediapackage element", e);
       return Response.serverError().status(Status.INTERNAL_SERVER_ERROR).build();
     } finally {
       IOUtils.closeQuietly(in);
```

Comment:
for these logs, it would also be useful to include the affected mpId.

Reply:
Not sure we have an mpId accessible here